streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -727,4 +734,17 @@ private void validateIsRunning() {
             throw new IllegalStateException("KafkaStreams is not running. State is " + state + ".");
         }
     }
+
+    /**
+     * Returns information about the local stream threads running in a {@link KafkaStreams} application.
```

Comment:
Nit: maybe `Returns runtime information about the local threads of this {@link KafkaStreams} instance.`